package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Denormalizer for normalizr",
   "main": "lib/index.js",
   "files": [
+    "dist",
     "lib"
```

Comment:
Why one version ? Build process is preparing two different versions on purpose.

This is setup exactly as for example `normalizr` (or other node libraries). Your current version (only transpiled from ES6 to ES5) will be in `lib` as always. So library users will include it, in their npm projects like always. They can compile their own bundle, minify, ...

But `dist` will contain special UMD version of the library, that can be used directly in a browser - for example allowing it in JSBin playground via https://unpkg.com/. Thats why the file in `dist` is `denormalizer.min.js` and not `index.js`.